cypress/integration/cityDetailsTest.js

```diff
@@ -0,0 +1,19 @@
+describe('City Details Page', () => {
+
+  beforeEach(() => {
+    cy.fixture('singleCityDetails.json').as('city').then((city) => {
+      cy.intercept('GET', 'https://softwhere-api.herokuapp.com/cities/3', {body: city});
+    });
+    cy.visit("http://localhost:3000/3");
+  });
+
+  it('User should still see the app header with the title and description', () => {
+    cy.get('header').first().contains('softWhere?');
+    cy.get('header').last().contains('Best US Cities to Live as a Software Developer');
+  });
+
+  it('User should see details about a single city', () => {
+    cy.get().contains('Raleigh');
+  });
+
```

Comment:
I'll admit, I'm still a bit confused on how much testing we are expected to do for each path.  In a job scenario, I can imagine testing that every `<p>` matches up, but I believe that the biggest expectation is that we are testing our fetch calls.